resources/design-document.md

```diff
@@ -0,0 +1,335 @@
+prospective cases, tracking upcoming deadlines, and providing a variety of interesting statistics that will help the 
+translator to improve productivity.
+
+## 2. Top Questions to Resolve in Review
+
+_List the most important questions you have about your design, or things that you are still debating internally that 
+you might like help working through._
+
+1. I would like to soft-delete translation clients and translation cases - would I still use DELETE in the endpoint? 
+2. 
+3.
+
+## 3. Use Cases
+
+#### Translation Cases
+
+U1. As a customer, I want to be able to view details about a translation case in a separate webpage.
+
+U2. As a customer, I want to be able to create a new translation case and either associate it with an existing 
+  translation client or create a new translation client at the same time.
+
+U3. As a customer, I want to be able to store relevant details about my translation case, such as cost per character, 
+  total payment, and whether the case is still open.
+
+U4. As a customer, I want to update existing translation cases with details about my translation progress (word count, 
+  dates, times).
+
+
+#### Translation Clients
+
+U5. As a customer, I want to be able to view details about a translation client in a separate webpage.
+
+U6. As a customer, I want to be able to edit translation client info.
+
+U7. As a customer, I want to be able to add a new translation case from the translation client detail page.
+
+U8. As a customer, I want to see a clickable list of all translation cases associated with a translation client, 
+  sorted by most recent activity.
+
+U9. As a customer, I want to see a clickable list of all translation cases associated with a translation client, sorted 
+  reverse-chronologically by creation date.  
+
+#### Home Page
+
+U10. As a customer, I want to see a list of open translation cases on the home page, sorted by soonest deadline.
+
+U10. As a customer, I want to see a list of up to 10 translation clients on the home page, sorted by most recent activity.
+
+U11. As a customer, I want to add a new translation case from the home page.
+
+U12. As a customer, I want to search for a translation client from the home page.
+
+U13. As a customer, I want to search for a translation case from the home page.
+
+U14. As a customer, I want to click a link on the home page to see a clickable list of all translation clients sorted alphabetically.
+
+U15. As a customer, I want to click a link on the home page to see a clickable list of all translation clients sorted by most recent activity.
+
+U16. As a customer, I want to click a link on the home page to see a clickable list of all translation cases associated with a certain type of project.
+
+U17. As a customer, I want to click a link on the home page to see a clickable list of all translation cases sorted by most recent activity.
+
+
```

Comment:
This feels like a lot of use cases, before we even get to stretch goals. I think it's doable to get through all of them, but would suggest prioritizing just in case you don't.